src/app/components/Notification/Notification.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import AppConfigStore from '../../stores/AppConfigStore';
+
+const Notification = (props) => {
+  const appConfigState = AppConfigStore.getState();
+  const {
+    notificationType,
+  } = props;
+
+  const notification = appConfigState[notificationType];
+
+  if (!notification) return null;
+
+  return (
+    <div className="nypl-banner-alert">
+      <p style={{ padding: '0px 20px 0px', margin: 0 }} dangerouslySetInnerHTML={{__html: notification }} />
```

Comment:
Don't care in this case, but in general I think it's best to keep CSS in the appropriate asset files. But not important for this change right now.